src/frame/system.rs

```diff
@@ -101,6 +101,8 @@ pub trait System {
         + MaybeSerialize
         + Debug
         + MaybeDisplay
+        + Encode
+        + Decode
```

Comment:
I think these are redundant, they come with `Paramater`

Reply:
q: `Parameter` also comes with `fmt::Debug`, so should I drop that too?